blueflood-core/src/main/java/com/rackspacecloud/blueflood/service/CoreConfig.java

```diff
@@ -156,6 +156,10 @@
     TENANTIDS_TO_KEEP(""),
     TRACKER_DELAYED_METRICS_MILLIS("300000"),
 
+    //the granulariy for which we store delayed metrics. Allowed values are 5m, 20m, 60m, 240m, 1440m
```

Comment:
"the granularity for which we store delayed metrics"
Does this mean we only store delay metrics for granularity 20m ?